test/compositemark/boxplot.test.ts

```diff
@@ -915,4 +915,35 @@ describe('normalizeBoxIQR', () => {
       }
     });
   });
+
+  it('should include necessary transform for joinaggregating in filtered layers', () => {
```

Comment:
should include necessary transform for the groupby fields of joinaggregate in filtered layers